html/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.0.3/socket.io.js"></script>
+<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?skin=sunburst"></script>
+
+<script>
+    function scrape() {
+        size = $('#size')[0].value;
+        $.get('/populateList', {size: size});
+        $("#process")[0].style.visibility = 'visible';
+    }
+    socket = io(window.location.href);
+
+    socket.on('done', function (data) {
+        $("#result")[0].innerText = data;
+        $("#result").removeClass("prettyprinted");
+        $("#process")[0].style.visibility = 'hidden';
+        PR.prettyPrint()
+    })
+</script>
+<h1>
+    UBC Courses Data Collector:
+</h1>
+<p>size: </p>
+<input id="size"/>
+<p>search: </p>
+<input id="search"/>
+<button onclick="scrape();">Click me!</button>
```

Comment:
`onclick` tags are generally considered bad practice--you can read up on [Unobtrusive Javascript](http://blog.teamtreehouse.com/unobtrusive-javascript-important)

Essentially, you want to separate the layout of the page (which is done via HTML) from the behaviours/interactions (which is done via JS)